recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+    name: pyamg
+    version: "3.0.2"
+
+source:
+    git_url: https://github.com/pyamg/pyamg.git
+    git_tag: v3.0.2
+
+build:
+    number: 2
+    string: np{{CONDA_NPY}}py{{CONDA_PY}}_{{PKG_BUILDNUM}}
```

Comment:
We don't encourage the use of build strings. Do you have a strong feeling to add that?